src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,27 @@
-export const TodoList = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { TodoInfo } from '../TodoInfo';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/types';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map(todo => (
+        <article
+          key={todo.id}
+          data-id={todo.id}
+          className={cn('TodoInfo', { 'TodoInfo--completed': todo.completed })}
+        >
+          <TodoInfo todo={todo} />
+
+          <UserInfo todo={todo} />
```

Comment:
Ensure that the `UserInfo` component receives the correct props. According to the checklist, data should be prepared in one place (App component) and passed to child components. Avoid using the `import` keyword in non-root components.